app/sagas/deepLinking.js

```diff
@@ -42,9 +41,12 @@ const handleOpen = function* handleOpen({ params }) {
 		host = host.slice(0, host.length - 1);
 	}
 
+	const { serversDB } = database.databases;
+	const currentServer = serversDB.objects('servers').filtered('currentServer = true');
```

Comment:
do `const { id: server, user } = yield serversDB.objects('servers').filtered('currentServer = true')[0];` and remove next lines where you are defining `const [server, user] =`